onyo/lib/inventory.py

```diff
@@ -124,15 +124,15 @@ def __init__(self, repo: OnyoRepo) -> None:
 
     @property
     def root(self):
-        """Path to root inventory directory"""
+        r"""Path to root inventory directory"""
```

Comment:
Trailing `.` missing everywhere in this file.

Reply:
Thanks for the catch. I have fixed it locally, but will not include it in this PR.